README.md

```diff
@@ -1,115 +1,107 @@
-import com.sinch.sdk.models.Configuration;
-
-...
-Configuration configuration = Configuration.builder().setKeyId(PARAM_KEY_ID)
-                                                     .setKeySecret(PARAM_KEY_SECRET)
-                                                     .setProjectId(PARAM_PROJECT_ID)
-                                                     .build();
-...
-SinchClient client = new SinchClient(configuration);
-```
-
-## Products
-
-Sinch client provides access to the following Sinch products:
-- [Numbers](https://developers.sinch.com/docs/numbers)
-- [SMS](https://developers.sinch.com/docs/sms) (WIP)
-
-Usage example of the `numbers` product:
-
-```java
-AvailableNumberListResponse response = client
-                                        .numbers()
-                                        .available()
-                                        .list(AvailableNumberListAllRequestParameters.builder()
-                                                                                    .setRegionCode("US")
-                                                                                    .setType(NumberType.LOCAL)
-                                                                                    .build());
-```
-
-## Logging
-The SDK uses the Java 8 logging feature ([java.util.logging](https://docs.oracle.com/javase/8/docs/api/java/util/logging/package-summary.html))
-Loggers and severity can be configured by using a `logging.properties` file like (see sample-app/src/main/resources/:
-```
-handlers = java.util.logging.ConsoleHandler
-java.util.logging.ConsoleHandler.level = INFO
-com.sinch.sdk.level = INFO
-
-java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
-java.util.logging.SimpleFormatter.format=[%1$tF %1$tT] [%4$-7s %2$s] %5$s %n
-```
-
-## Onboarding/Examples
-The [sample-app](sample-app/README.md) directory contains ready to use samples and can be used to start your own applications based onto Sinch SDK.
-
-## HTTP Client
-See [HTTP-CLIENT](HTTP-CLIENT.md) file
-
-## Architecture
-See [ARCHITECTURE](ARCHITECTURE.md) file
-
-## License
-
-This project is licensed under the Apache License. See the [LICENSE](LICENSE) file for the license text.
\ No newline at end of file
+# Sinch Java SDK
+
+Here you'll find documentation related to the Sinch Java SDK, including how to install it, initialize it, and start developing <language> code using Sinch services.
+
+To use Sinch services, you'll need a Sinch account and access keys. You can sign up for an account and create access keys at [dashboard.sinch.com](https://dashboard.sinch.com).
+
+For more information on the Sinch APIs on which this SDK is based, refer to the official [developer documentation portal](https://developers.sinch.com).
+
+**This SDK is currently available to selected developers for preview use only. It is being provided for the purpose of collecting feedback, and should not be used in production environments.**

```

Comment:
Side note, who is the `selected developers`? Can we just mention that it's prerelease so prod use is not recommended and a possibility of breaking changes to come?

Reply:
@alex-sberna : What do you think about ?

Reply:
I think this is a good idea. We can probably even just copy the same note that's on developers.sinch.com: "This SDK is currently available for preview purposes only. It should not be used in production environments."